frontend/src/scenes/max/maxContextLogic.ts

```diff
@@ -585,11 +594,42 @@ export const maxContextLogic = kea<maxContextLogicType>([
             },
         ],
     }),
-    afterMount(({ cache }) => {
+    afterMount(({ cache, actions }) => {
         cache.previousLocation = {
             location: router.values.location,
             hashParams: router.values.hashParams,
             searchParams: router.values.searchParams,
         }
+        actions.registerTool({
```

Comment:
I would put this logic somewhere else, maybe `maxGlobalLogic`, and keep this one only for UI context.

Reply:
I actually moved it from `maxGlobalLogic` to here, but makes sense to be opinionated here – moved back